frontend/src/search-facets.ts

```diff
@@ -73,6 +73,30 @@ export class SearchaliciousFacets extends SearchActionMixin(
     });
   }
 
+  /**
+   * Get a facet node by its taxonomy
+   * It will return undefined if the taxonomy is not found
+   * @param taxonomy
+   */
+  getFacetNodeByTaxonomy(taxonomy: string): SearchaliciousFacet | undefined {
+    return this._facetNodes().find((node) => node.taxonomy === taxonomy);
```

Comment:
If I look at bellow taxonomy is a function not a property. Am I wrong ?

```suggestion
    return this._facetNodes().find((node) => node.taxonomy() === taxonomy);
```